src/views/LoggingRoom/LoggingRoomCreate.vue

```diff
@@ -3,9 +3,12 @@
       <div class="logging-room-create__container">
         <BaseCard class="logging-room-create__content">
           <div class="logging-room-create__save-button">
-            <BaseButton class="save-btn" type="submit" buttonClass="button-save" @click="saveLoggingRoom">
-              <font-awesome-icon class="icon" icon="save"/> Save
-            </BaseButton>
+            <b-button class="is-rounded is-primary  save-btn"
+                      type="submit" buttonClass="button-save"
+                      icon-left="save"
+                      @click="saveLoggingRoom">
+              <span>Save</span>
+            </b-button>
           </div>
           <div class="logging-room-create__input-title">
             <BaseInput :label="titleLabel" class="input-title" v-model="titleTemp"></BaseInput>
```

Comment:
BaseInput is a deprecated component, can you please change this to use `b-input` instead?